src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,195 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Scanner;
+
+public class Main {
+
+    public static final String INFO_CREATE
+            = "create [path] [file-name] - Создает текстовый файл по указанному пути;";
+    public static final String INFO_READ
+            = "read [path] [file-name] - Считывает файл по указанному пути и "
+            + "выводит текст в консоль;";
+    public static final String INFO_INFO
+            = "info [path] [file-name] - Выводит краткую информацию по указанному файлу;";
+    public static final String INFO_HELP
+            = "help - Выводит в консоль все доступные команды и информацию к ним;";
+    public static final String INFO_HELP_COMMAND
+            = "help [command] - Выводит в консоль информацию по указанной команде;";
+    public static final String INFO_EXIT
+            = "exit - Завершение работы программы.";
+
+    public static void chooseCommand(String input) {
+        String[] command = input.split(" ");
+        switch (command[0]) {
+            case "create":
+                create(new File(command[1], command[2]));
+                break;
+            case "read":
+                System.out.println("Содержимое файла " + "\'" + command[2] + "\' :");
+                System.out.print(read(new File(command[1], command[2])));
+                break;
+            case "info":
+                info(new File(command[1], command[2]));
+                break;
+            case "help":
+                help(command);
+                break;
+            case "exit":
+                System.exit(0);
+                break;
+            default:
+                noCommand(input);
+                break;
+        }
+    }
+
+    public static void create(File file) {
```

Comment:
@vladkuzya No business logic in the Main class. The Main class is only the entry point. Move this method to the FileService class